### ISSUE TITLE
Always format when selecting "Format" command from palette

### ISSUE THREAD

**Issue by jhildenbiddle** [prettier/prettier-vscode#1777]
**Is your feature request related to a problem? Please describe.**

TL;DR: Selecting a "Format" command from the command palette should ignore the "Prettier: Require Config" setting.

I work on multiple projects, some that use prettier and some that do not. When working on projects that use prettier, the "Editor: Format On Save" option is very handy. This option can be problematic when working on projects that do not user prettier though (any file changes causes a reformat of the entire file), so I have also enabled the "Prettier: Require Config" option. So far, so good.

The problem with this setup is that it prevents me from formatting anything when working on projects that do not use prettier. Manually invoking a "Format" command from the command palette results in no change, presumably because the "Prettier: Require Config" is cancelling the call. I would expect manually invoking a "Format" command to override any option related to auto-formatting.

**Describe the solution you'd like**

Ideally, I just want formatting to work 1) automatically if there's a prettier config and 2) when manually called via the command palette *regardless* of my "Prettier: Require Config" setting. I don't want to have formatting functionality disabled completely.

A few ideas on how to accomplish this:

* Modify the "Prettier: Require Config" option so that it only applied to automated formatting (on save, on paste, etc). 
* Provide a "Prettier: Always format from command palette" option
* Provide a "Prettier: Fallback Formatter" option that will invoke an alternate formatter when 1) "Prettier: Require Config" is enabled, 2) a prettier config is not found, and 3) when the user selects a "Format" command from the command palette 

**Describe alternatives you've considered**

Disabling format-on-save option, which is not really what I want to do.

**Comment by ntotten** [comment]
Duplicate of #1672

**Comment by jhildenbiddle** [comment]
Thanks, @ntotten. Thought I did a pretty good scan of the open issues before creating this one, but apparently that wasn't the case. :)
